cmd/envtool/tests.go

```diff
@@ -486,7 +486,10 @@ func runGoTest(runCtx context.Context, opts *runGoTestOpts) (resErr error) {
 				msg += fmt.Sprintf(" %d/%s", done, totalTests)
 			}
 
-			logOutput := opts.logger.WarnContext
+			logOutput := opts.logger.ErrorContext
+			if event.Action == "skip" {
+				logOutput = opts.logger.WarnContext
+			}
```

Comment:
That's just a spot improvement to use different colors (yay!) for skipped and failing tests.

Reply:
Please remove it from that PR

Reply:
ok, let me do that.

Reply:
#4974